src/core/requests/HttpRequest.js

```diff
@@ -7,6 +7,9 @@ export default class HttpRequest {
 
```

Comment:
Ta konfiguracja jest taka sama dla wszystkich metod. Jedyna różnica to nazwa metody. Zmiany:
1.Przenieść tworzenie obiektu options to metody execute.
2.Dodać do metody execute parametr z nazwą metody zamiast obiektu options
3.W metodzie execute dodawać header Authorization tylko wtedy, gdy jest dostępny